aten/src/ATen/native/cuda/Normalization.cu

```diff
@@ -722,6 +722,16 @@ std::tuple<Tensor, Tensor> batch_norm_update_stats_cuda(
   c10::MaybeOwned<Tensor> running_var = at::borrow_from_optional_tensor(running_var_opt);
 
   const int64_t n_input = self.size(1);
+
+  TORCH_CHECK(
+      self.size(0) > 0 && n_input > 0 && self.numel() > 0,
+      "input.size(0), input.size(1) and input.numel() need to be greater than 0, got input.size(0) = ",
+      self.size(0),
+      ", input.size(1) = ",
+      n_input,
+      ", input.numel() = ",
+      self.numel()
+  );
```

Comment:
Will there be the same issue as https://github.com/pytorch/pytorch/issues/120803 on CUDA ?

Reply:
```
import torch
input = torch.rand(0, 1)
input.to("cuda")
running_mean = torch.rand(1)
running_var = torch.rand(1)

running_mean.to("cuda")
running_var.to("cuda")
torch.batch_norm_update_stats(input=input, momentum=0.0, running_mean=running_mean, running_var = running_var)
```

gives
```
[1]    21652 floating point exception  /home/lancerts/miniconda3/envs/wsl-dev/bin/python
```

Reply:
Got it. Thanks !